controllers/auth.ctrl.go

```diff
@@ -42,6 +52,20 @@ func (controller *AuthController) Auth(c echo.Context) error {
 		return c.JSON(http.StatusBadRequest, responses.BadArgumentsError)
 	}
 
+	if body.Login == "" || body.Password == "" {
```

Comment:
In Go the body and the form params are different things. When I was testing the authentication in the swagger UI, the body was empty because the data was sent as form-data. Apparently Bluewallet did not follow the [Oauth2Password](https://fastapi.tiangolo.com/tutorial/security/simple-oauth2/) spec precisely, or maybe they didn't mean to. Since it is very convenient if you are able to do this in the swagger web UI, I thought this was a good idea to check in the form params as well.